test/inductor/test_cpu_select_algorithm.py

```diff
@@ -417,7 +429,7 @@ def forward(self, x):
                 equal_nan=True,
                 exact_dtype=True,
             )
-            self.assertEqual(counters["inductor"]["select_algorithm_autotune"], 1)
+            self.assertEqual(counters["inductor"]["select_algorithm_autotune"], 2)
```

Comment:
why?
Also, please add check: `self.assertEqual(counters["inductor"]["cpp_epilogue_fusion_counter"], 1)`

Reply:
The testcase has been changed from single Linear module to 2 nearby Linear modules. That will hit the autotuning twice.
Thanks and added the check of `self.assertEqual(counters["inductor"]["cpp_epilogue_fusion_counter"], 0)`, since the `relu`, `gelu` has been fused into `onednn.qlinear` as part of template instead of fusion by scheduler.